assets/startup.sh

```diff
@@ -24,24 +21,16 @@ rm -f /var/run/chrony/chronyd.pid
 
 
 # NTP_SERVERS environment variable is not present, so populate with default server
-if [ -z ${NTP_SERVERS} ]; then
-  echo "server ${DEFAULT_NTP} iburst" >> ${CHRONY_CONF_FILE}
-
-# check if list of ntp servers provided in NTP_SERVERS environment variable
-# are present and is of greater than length 0.
-elif [ $(echo ${#NTP_SERVERS}) > 0 ]; then
-  IFS=","
-  for N in $NTP_SERVERS; do
-    # strip any quotes found before or after ntp server
-    echo "server "${N//\"}" iburst" >> ${CHRONY_CONF_FILE}
-  done
-
-# NTP_SERVERS environment variable is present, but doesn't contain ntp servers, so
-# populate with a default server.
-else
-  echo "server ${DEFAULT_NTP} iburst" >> ${CHRONY_CONF_FILE}
+if [ -z "${NTP_SERVERS}" ]; then
+  NTP_SERVERS="${DEFAULT_NTP}"
 fi
 
+IFS=","
+for N in $NTP_SERVERS; do
+  # strip any quotes found before or after ntp server
+  echo "server "${N//\"}" iburst" >> ${CHRONY_CONF_FILE}
+done
+
```

Comment:
this passes the three cases i test for:

 1. no `NTP_SERVERS` 
 2. empty `NTP_SERVERS` list
 3. custom `NTP_SERVERS` list

looks good 👍